airflow/models/serialized_dag.py

```diff
@@ -20,11 +20,12 @@
 
 import hashlib
 import logging
+import zlib
```

Comment:
Without meaning to start a compression format war: Is zlib really an appropriate choice?

The python stdlib includes `bz2` which can produce blobs as much as 50% smaller -- though obviously that depends upon the data https://starbeamrainbowlabs.com/blog/article.php?article=posts%2F275-Compression-Comparison.html

Reply:
although `bz2` can achieve smaller size of compress data, but we noticed that the `zlib` is faster than `bz2` when  compressing and decompressing data with the same level of compression (zlib with default level 6, also set compresslevel as 6 for bz2).

test with python3.7.9, `raw json data size`: `514MB`, `bz2` compressed: `14M` and `zlib` compressed: `44M`.

```
ping_zhang@i-001e445e8c0ed963f:~$ python test_bz2.py
bz2 took 89.23221111297607
bz2 decompress took 15.989518880844116


ping_zhang@i-001e445e8c0ed963f:~$ python test_zlib.py
zlib compress took 8.829093933105469
zlib decompress took 1.1581130027770996

```

Reply:
Oh yeah, 15s to decompress clearly makes bz2 unsuitable!

(I knew bz2 was slow to compress, but I mistakenly thought it was fast to decompress too.